custom_module/cypher_queries/db_management_queries.py

```diff
@@ -0,0 +1,17 @@
+from promg import Query
+
+class DBManagementQueries:
+    @staticmethod
+    def create_db(dbname):
+        query_str = '''CREATE OR REPLACE DATABASE $dbname'''
+        return Query(query_str=query_str, template_string_parameters={"dbname": dbname} )
+
+    @staticmethod
+    def drop_db(dbname):
```

Comment:
Since these function return queries, I use the naming convention get_what does it do_query()

```suggestion
    def get_drop_db_query(dbname) -> Query:
```